src/pluto/grayscale_image.cr

```diff
@@ -54,10 +54,22 @@ class Pluto::GrayscaleImage < Pluto::Image
   end
 
   def to_rgba
-    RGBAImage.new(@gray.clone, @gray.clone, @gray.clone, Array(UInt8).new(size) { 1u8 }, width, height)
+    RGBAImage.new(@gray.clone, @gray.clone, @gray.clone, Array(UInt8).new(size) { 255u8 }, width, height)
```

Comment:
Found a bug in my first implementation :P Though since pluto doesn't support PNG yet, the alpha channel hasn't impacted anything yet.